flexget/utils/imdb.py

```diff
@@ -37,6 +37,26 @@ def is_imdb_url(url):
     return re.match(r'https?://[^/]*imdb\.com/', url)
 
 
+def is_valid_imdb_title_id(value=None):
+    """
+    Return True if `value` is a valid IMDB ID for titles (movies, series, etc).
+    """
+    if not isinstance(value, basestring):
+        return False
```

Comment:
I'm now raising a `TypeError` in case `value` is not a string. However, I think it is better to not raise an exception and instead return `False` if `value` is `None`. The reason is that we can write something as

```
if is_valid_imdb_title_id(entry.get('nfo_id')):
    do_some_work_using_nfo_id_field()
```

In case the entry does not have the field then `entry.get` returns `None`,
`is_valid_imdb_title_id` returns `False` and we don't try to use the field
value. If `is_valid_imdb_title_id` raises an exception for `None` then everyone would have to test for that.